libraries/ESP_NOW/examples/ESP_NOW_Broadcast_Master/ESP_NOW_Broadcast_Master.ino

```diff
@@ -0,0 +1,126 @@
+/*
+    ESP-NOW Broadcast Master
+    Lucas Saavedra Vaz - 2024
+
+    This sketch demonstrates how to broadcast messages to all devices within the ESP-NOW network.
+    This example is intended to be used with the ESP-NOW Broadcast Slave example.
+
+    The master device will broadcast a message every 5 seconds to all devices within the network.
+    This will be done using by registering a peer object with the broadcast address.
+
+    The slave devices will receive the broadcasted messages and print them to the Serial Monitor.
+*/
+
+#include "ESP32_NOW.h"
+#include "WiFi.h"
+
+#include <esp_mac.h> // For the MAC2STR and MACSTR macros
+
+/* Definitions */
+
+#define ESPNOW_WIFI_CHANNEL 6
+
+/* Classes */
+
+// Create a new class that inherits from the ESP_NOW_Peer class is required to implement the _onReceive and _onSent methods.
+
+class ESP_NOW_Broadcast_Peer : public ESP_NOW_Peer {
+public:
```

Comment:
Since this class is in the sketch itself, you can implement the methods directly inside. No need for separate declaration and implementation